File: assets/samples/brand.alert/python/brand_alert_api_example.py
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

terms = 'cinema';
password = 'Your whois api password'
username = 'Your whois api username'
rows = '5'

url = 'http://www.whoisxmlapi.com/brand-alert-api/search.php?'\
    + 'term1=' + terms + '&username=' + username + '&password='\
    + password + '&rows=' + rows

print(urlopen(url).read().decode('utf8'))

File: assets/samples/brand.alert/php/brand_alert_api_example.php
<?php
    $terms = 'cinema';
    $password = 'Your whois api password';
    $username = 'Your whois api username';
    $rows = '5';
    $url = "https://www.whoisxmlapi.com/brand-alert-api/search.php?"
        ."term1={$terms}&username={$username}&password={$password}&rows={$rows}";

    print(file_get_contents($url));